### DISCUSSION TITLE
🕰️ Longevity 🕰️

### DISCUSSION THREAD

**Discussion by bschilder** [neurogenomics/rworkflows#93] [Ideas]
Times change, personnel turn over, and priorities shift over time. 
# So the question becomes; how do we ensure the longevity of open-source projects like `rworkflows`? 🕰️

Here we discuss all topics towards ensuring the continued maintenance and improvement of `rworkflows` well into the future! 🚀

The goal of this Discussion is to invite all members of the community to share ideas/concerns/avenues/experiences that may contribute the longevity of `rworkflows` (or other open-source projects like it) in order to maximise and sustain its usefulness to all stakeholders.

![](https://media.giphy.com/media/O76l7iUl5BeOA/giphy.gif)

**Comment by bschilder** [comment]
# Related Issues

- #92
- #63 
- #94

**Comment by farsoroush** [comment]
A procedure to ensure 6months+ overlap between team member changes, so the incoming and outgoing members have some time to communicate.
